onboarder/installation/index.rst

```diff
@@ -79,10 +79,12 @@ Here are two examples in order to define environment variables:
 
 .. warning::
 
-    All the following variables must be set in order to configure the Onboarder correctly for all entrypoints, all processes that runs the PIM code source.
+    All the following variables must be set in order to configure the Onboarder correctly for all entrypoint, all processes that runs the PIM code source.
```

Comment:
why?

Reply:
According to PHPStorm dictionary, it is invariable. I'm gonna check on Language Tools.

Reply:
OK, so in fact, according to LanguageTool, Grammarly and Antidote, this word just doesn't exist :laughing:. It is always in 2 words: "entry points".